src/patterns/language-selection/index.njk

```diff
@@ -0,0 +1,42 @@
+---
+title: Change Language
+group: Help users to...
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+
+## When to use this pattern
+
+This pattern should be if you service is available in multiple languages.
+
+## How to use this pattern
+
+Language selection is achieved by rendering a language selector component in the header element, and a list of available lanauges in the footer. The header lanauge selector is for viewports larger than mobile, and the footer language selector is for viewports smaller than tablet.
+
+If there are 3 or less available languages, the header language picker will render two links, otherwise a dropdown menu will be renedered.
+
+## Examples
+
+### 3 Available languages
+{{
+    patternlibExample({ "path": "patterns/language-selection/examples/english-ireland/index.njk", "useIframe": true, "iframeHeight": 400 })
+}}
+
+### 2 Available languages
+{{
+    patternlibExample({ "path": "patterns/language-selection/examples/cymraeg/index.njk", "useIframe": true, "iframeHeight": 400 })
+}}
+
+### 5 Available languages
+{{
```

Comment:
What happens when the user selects 'all languages'?

Reply:
It would link to a separate page with a list of all languages, although we won't be using that for a while. If the all languages link isn't supplied it shouldn't render the all languages option so maybe I should do that in all the examples until we take on loads of languages.

Reply:
Yeah leave it off the example for now then.  I'm just trying to think of questions designers might ask when reading this.

Reply:
I've removed the all languages options for now... we can add them in at a later point when we take on more than say 5 languages